### PR TITLE
Feature/WLZAA-41 add Home special component

### PR DESCRIPTION
**Pull request by susomejias**
# Don't forget to add Jira issue related code to the title 😇

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix

## Description

- Add Home special component

### Mobile

demo-special-mobile

### Tablet

demo-special-tablet

### Desktop

demo-special-desktop

### CODE REVIEW

File: src/theme/color.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export const PRIMARY = '#f06078'
 export const ACCENT = '#6666ff'
 export const DARK_GRAY = '#3c3c3b'
-export const LIGHT_GRAY = '#f5f5f5'
+export const LIGHT_GRAY = '#<f5f5f5>'
```
Comment (DrankoLQ) [inline src/theme/color.ts:4]: Uh, eso es un typo?

**Review by DrankoLQ — APPROVED** [review]
💅

File: src/theme/color.ts (hunk 1 above)
Comment (susomejias) [reply src/theme/color.ts:4]: WTF se ha colado no sé cómo?? ajajaja

Comment (susomejias) [reply src/theme/color.ts]: happy

**Review by Miangame — APPROVED** [review]
Mu rico!

**Pull request merged**